### PR TITLE
Update sl-partner-integration-guide.md

### PR DESCRIPTION
**Pull request by rpourzand**
## What are you changing in this pull request and why?

Adding a note on saved queries to our integration guide

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/guides/sl-partner-integration-guide.md [hunk 1]
```diff
@@ -142,6 +148,7 @@ These are recommendations on how to evolve a Semantic Layer integration and not
 * Listing available dimensions based on one or many metrics
 * Querying defined metric values on their own or grouping by available dimensions
 * Display metadata from [Discovery API](/docs/dbt-cloud-apis/discovery-api) and other context
+* Use [saved queries](/docs/build/saved-queries) to query pre-build metrics, dimensions, and filters. Refer to the [JDBC](/docs/dbt-cloud-apis/sl-jdbc) and [GraphQL](/docs/dbt-cloud-apis/sl-graphql) APIs for syntax.
```
Comment (mirnawong1) [inline website/docs/guides/sl-partner-integration-guide.md]: @rpourzand also added this here as integration recommendations - do you think it makes sense?

**Review by mirnawong1 — APPROVED** [review]
updated the bullet format and added linnks - lgtm but have an outstanding question for @rpourzand !

File: website/docs/guides/sl-partner-integration-guide.md (hunk 1 above)
Comment (rpourzand) [reply website/docs/guides/sl-partner-integration-guide.md]: Yeah looks great! I might tweak it slightly to read like 

"Expose "Saved Queries" which are pre-built metrics, dimensions, and filters that Semantic Layer developers create for easier analysis. You can expose them in your application - refer to the JDBC and GraphQL APIs..."

Comment (mirnawong1) [inline website/docs/guides/sl-partner-integration-guide.md]:
```suggestion
* Expose [Saved queries](/docs/build/saved-queries), which are pre-built metrics, dimensions, and filters that Semantic Layer developers create for easier analysis. You can expose them in your application. Refer to the [JDBC](/docs/dbt-cloud-apis/sl-jdbc) and [GraphQL](/docs/dbt-cloud-apis/sl-graphql) APIs for syntax.
```

**Comment by mirnawong1** [comment]
merging this pr now @rpourzand !

**Pull request merged**